ocs_ci/helpers/helpers.py

```diff
@@ -5433,6 +5435,42 @@ def update_volsync_channel():
             )
 
 
+def update_volsync_icsp():
+    """
+    Update Volsync ImageContentSourcePolicy.
+
+    """
+    logger.info("Creating ImageContentSourcePolicy for Volsync")
+    restore_index = config.cur_index
+    non_acm_clusters = get_non_acm_cluster_config()
+    for non_acm_cluster in non_acm_clusters:
+        index = non_acm_cluster.MULTICLUSTER["multicluster_index"]
+        config.switch_ctx(index)
+        run_cmd(f"oc create -f {constants.ACM_BREW_ICSP_YAML}")
+        wait_for_machineconfigpool_status("all", timeout=1800)
+        volsync_pod_obj = get_pods_having_label(
+            namespace="volsync-system", label="app.kubernetes.io/name=volsync"
+        )[0]
```

Comment:
can we avoid directly indexing the list here?

Reply:
fixed in new commit